tests/whitebox/integration/test_parser.py

```diff
@@ -18,8 +18,29 @@
 # isort: STDLIB
 import unittest
 
+# isort: LOCAL
+from stratis_cli._error_reporting import StratisCliErrorCodes
+
 from ._misc import RUNNER, SimTestCase
 
+_PARSE_ERROR = StratisCliErrorCodes.PARSE_ERROR
+
+
+def check_parse_error(obj, command_line, expected_code):
+    """
+    Check that running the program with given prefix and command line arguments
+    will return an exit code which matches the expected code, in this case a
+    parser error.
+
+    :param command_line: the command line arguments
+    :param expected_code: the expected exit code
+    :type expected_code: int
+    """
+    with obj.assertRaises(SystemExit) as context:
+        RUNNER(command_line)
+    exit_code = context.exception.code
+    obj.assertEqual(exit_code, expected_code)
```

Comment:
You should hard-code the ```expected_code``` as ```PARSE_ERROR```, I think. In this case, the method is crafted just for these exceptions, handled in this way.